test/svgo-to-vectordrawable-test.js

```diff
@@ -47,4 +47,19 @@ describe('svg-to-vectordrawable', function() {
                 '</vector>\n')});
     });
 
+    describe('Stop offset default value support', () => {
```

Comment:
Seems like async/await should be ok since node >= 8.0.0 specified in package.json.